dm/dm-precheck.md

```diff
@@ -90,6 +90,30 @@ For the full data migration mode (`task-mode: full`), in addition to the [common
 
     - If sharded tables have auto-increment primary keys, the precheck returns a warning. If there are conflicts in auto-increment primary keys, see [Handle conflicts of auto-increment primary key](/dm/shard-merge-best-practices.md#handle-conflicts-of-auto-increment-primary-key) for solutions.
 
+#### Check items for physical import
+
+If you set `import-mode: "physical"` in the task configuration, the following check items are added to ensure that [Physical Import](/tidb-lightning/tidb-lightning-physical-import-mode.md) runs normally. After following the prompts, if you find it difficult to meet the requirements of these check items, you can try to use the [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) to import data.
+
+* Empty Regions in the downstream database
+
+    - If the number of empty Regions is greater than `max(1000, 3 * the number of tables)` (the larger of "1000" and "3 times the number of tables"), the precheck returns a warning. You can adjust related PD parameters to speed up the merging of empty Regions and wait for the number of empty Regions to decrease. See [PD Scheduling Best Practices - Slow Region Merge](/best-practices/pd-scheduling-best-practices.md#region-merge-is-slow).
+
+* Region distribution in the downstream database
+
+    - Checks the number of Regions in different TiKV nodes. If the node with the least number of Regions has a number of Regions that is 75% of the node with the most number of Regions, the precheck returns a warning. You can adjust related PD parameters to speed up the scheduling of Regions and wait for the number of Regions to change. See [PD Scheduling Best Practices - Leader/Region distribution is not balanced](/best-practices/pd-scheduling-best-practices.md#leadersregions-are-not-evenly-distributed).
```

Comment:
> If the node with the least number of Regions has a number of Regions that is 75% of the node with the most number of Regions

hard to read 😂 can we change the subject to number (of regions on node) and refine it?

Reply:
Aggree. It's a bit hard to read. Can we state the meaning in this way? 
 (I am not sure whether it's correct) https://github.com/pingcap/docs-cn/pull/12296/files#r1095384413
![image](https://user-images.githubusercontent.com/41534398/216524214-e44c5c89-a169-4892-a7af-719f4656f5fa.png)